code/datums/ai/basic_mobs/basic_subtrees/attack_obstacle_in_path.dm

```diff
@@ -25,6 +25,8 @@
 	action_cooldown = 1 SECONDS
 	/// If we should attack walls, be prepared for complaints about breaches
 	var/can_attack_turfs = FALSE
+	/// Tries to bump open airlocks with an attack
+	var/bump_open_airlock = FALSE
```

Comment:
Would this be better as a component which picks up the mob attempting to attack a door and tries to open it instead? That seems like it'd be more exetendable.

Reply:
That probably would be better

Reply:
Got it